client/public/js/whiteboard.js

```diff
@@ -19,48 +30,58 @@ const newCanvas = function(){
 }
 
 const drawTouch = function() {
-	let start = function(e) {
-		ctx.beginPath();
-		x = e.changedTouches[0].pageX;
-		y = e.changedTouches[0].pageY - 34;
-		ctx.moveTo(x, y);
+
+	const startTouch = function(e) {
+    // if (e.targetTouches.length < 2) {
+      // e.preventDefault();
+  		ctx.beginPath();
+  		x = e.changedTouches[0].pageX + content.scrollLeft;
```

Comment:
`const x = ...` 

The variables `x` and `y` are never defined
`content` here is undefined unless I'm missing a global variable